src/ImageLoaders/Xbe/XbeLoader.cs

```diff
@@ -294,11 +297,11 @@ public override Program LoadProgram(Address? addrLoad)
                 segments.Add(tlsSegment);
             }
 
-            var imports = LoadImports();
+            SegmentMap segmentMap = new SegmentMap(ctx.EntryPointAddress, segments.ToArray());
```

Comment:
I'm not sure `ctx.EntryPointAddress` is appropriate here. The first parameter of the `SegmentMap` ctor expects the base address of the image, which isn't guaranteed to be the same as the entry point. If you don't specify the entry point address -- only passing segments -- Reko will calculate the base address as the `min` of the `ImageSegment`s' addresses.